lib/charty/table_adapters/hash_adapter.rb

```diff
@@ -123,7 +128,7 @@ def initialize(data, columns: nil, index: nil)
         arrays = arrays.map do |array|
           case array
           when Charty::Vector
-            array.data
+            array.data.to_a
```

Comment:
We should ensure converting to raw `Array` instead of using `Vector`'s data as `Array` like data.

Reply:
Like above.  I want to prevent converting the original vector types to `Array`.